### DISCUSSION TITLE
Kubernetes uses minio to create a storageclass

### DISCUSSION THREAD

**Discussion by saowu** [minio/minio#17211] [Q&A]
Cloud manufacturers have implemented attaching object buckets to pods as a persistence solution. How does Minio implement this?

**Comment by saowu** [comment]
There is no mini cluster installed within the cluster, using an existing external minio cluster

**Comment by vadmeste** [comment]
@saowu I am not sure I understand your question. Could you elaborate more ?

**Reply by saowu** [reply]
I currently have a Minio cluster and a Kubernetes cluster. Minio is not deployed in the Kubernetes cluster, and I want to mount and use the object bucket in Pod like nfs. I have searched for the relevant article, but it is too complex and I am not sure if there is a reliable solution.


http://www.lishuai.fun/2021/12/31/k8s-pv-s3/#/%E8%A6%81%E6%B1%82

**Reply by saowu** [reply]
Cloud vendor implementation documents
https://support.huaweicloud.com/usermanual-cce/cce_01_0160.html
https://help.aliyun.com/document_detail/134896.html?spm=a2c4g.127554.0.0.718c5912I1SEas
